src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage,
+  closeError: () => void,
+};
+
+export const Error: React.FC<Props> = ({
+  errorMessage,
+  closeError,
+}) => {
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      closeError();
+    }, 3000);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, []);
```

Comment:
You should add the `closeError` function to the dependency array of `useEffect` to avoid potential stale state issues. 🛠